src/components/camera-tool.js

```diff
@@ -254,9 +263,35 @@ AFRAME.registerComponent("camera-tool", {
         }
         renderer.readRenderTargetPixels(this.renderTarget, 0, 0, width, height, this.snapPixels);
         pixelsToPNG(this.snapPixels, width, height).then(file => {
-          const { entity, orientation } = addMedia(file, "#interactable-media", undefined, true);
-          entity.object3D.position.copy(this.el.object3D.position).add(new THREE.Vector3(0, -0.5, 0));
+          const { entity, orientation } = addMedia(file, "#interactable-media", undefined, false);
+
+          const pos = this.el.object3D.position;
+
+          entity.object3D.position.set(pos.x, pos.y, pos.z);
           entity.object3D.rotation.copy(this.el.object3D.rotation);
+          entity.object3D.rotateY(Math.PI);
+          entity.object3D.scale.set(0.1, 0.1, 0.1);
+
+          // Generate photos in a circle around camera, starting from the bottom.
```

Comment:
I like it! I would make the first photo go directly under the camera instead. Feels like a bug when you only take 1 photo.

Reply:
Ah I considered that but the problem with that is that unless I got fancier with the algorithm, it'd force placement of another one directly above the camera, behind the viewfinder. Basically the ideal arrangement isn't a linear polar walk but that's what I hacked out here